ingest/workflow/snakemake_rules/transform.smk

```diff
@@ -42,8 +42,8 @@ rule transform:
         all_geolocation_rules="data/all-geolocation-rules.tsv",
         annotations=config["transform"]["annotations"],
     output:
-        metadata="results/metadata.tsv",
-        sequences="results/sequences.fasta",
+        metadata="data/metadata.tsv",
+        sequences="data/sequences.fasta",
```

Comment:
On further reflection about my rule-splitting comment above, what if you didn't change the output directory here to `data/` and instead wrote out the metadata and sequences to `results/metadata_all.tsv` and `results/sequences_all.fasta`? Then you don't have to copy any files and you can drop the if/else logic from your `split_serotypes.smk` rule above.

Reply:
I agree! Fixed and your suggestion streamlines the pipeline 0537e23053972cdee9f2514769ef572143032477